packages/@adobe/spectrum-css-temp/components/inputgroup/skin.css

```diff
@@ -103,7 +103,7 @@ governing permissions and limitations under the License.
     specifically for readonly inputgroups that aren't disabled since the button will have the disabled class
     but we don't want the border color to be the disabled quiet one
   */
-  &:not(.is-disabled) {
+  &:not(.is-disabled):not(.is-focused) {
```

Comment:
what changed that introduced this bug though?

Reply:
the break happened after https://github.com/adobe/react-spectrum/pull/6777 which included a lightning css version minor bump. I'm guessing that was the culprit but I didn't dig much further than that